sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/transforms/RunInference.java

```diff
@@ -153,12 +184,17 @@ public PCollection<OutputT> expand(PCollection<?> input) {
       outputCoder = (Coder<OutputT>) KvCoder.of(keyCoder, RowCoder.of(schema));
     }
 
+    if (this.extraPackages.isEmpty()) {
+      this.extraPackages.addAll(mayBeInferExtraPackagesFromModelHandler());
```

Comment:
Shoud we print out the info log that we're adding automatically inferred extra packages?